onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -227,37 +222,32 @@ class AttentionCPUBase : public AttentionBase {
       present += SafeInt<ptrdiff_t>(batch_size) * num_heads_ * total_sequence_length * v_head_size;
     }
 
-    const double cost =
```

Comment:
This cost shall be sequence_length * v_head_size * total_sequence_length, which are m/n/k of the MatMul